### PR TITLE
Blog post for tech preview of dev mode with container support

### LINKED ISSUE #572: Blog post on tech preview for dev mode with containers
The purpose of this blog post is to announce the tech preview for [dev mode with containers](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#devc-container-mode) so that developers can try it out and give feedback.

### PR DESCRIPTION
**Pull request by ericglau**
Fixes #572

Draft can be viewed here: https://draft-openlibertyio.mybluemix.net/blog/2020/07/24/dev-mode-containers-preview.html

Please review content.

### CODE REVIEW

**Review by turkeylurkey — APPROVED** [review]

File: posts/2020-07-24-dev-mode-containers-preview.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+In this post, we introduce a technology preview for container support in Open Liberty dev mode.  This lets you develop applications on your local environment while your Open Liberty server runs in a container.  The development image is kept as similar as possible to the production image while allowing for iterative development, so that your code changes are automatically hot deployed to the container and picked up by the running server.  Additionally, dev mode lets you run tests either automatically or on demand, and you can attach a debugger at any time to debug your application.
+
+Note that since container support is in technology preview, its features and parameters are subject to change in future milestones or releases of the Liberty Maven or Gradle plugins.
```
Comment (yeekangc) [inline posts/2020-07-24-dev-mode-containers-preview.adoc]: Can we add a call to action here i.e. invite the users to try it out and provide feedback?  And, embed a video demo?

File: posts/2020-07-24-dev-mode-containers-preview.adoc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+== Additional resources
+
+For a more comprehensive demo, see the link:https://github.com/OpenLiberty/demo-devmode/tree/devc[devc branch of the demo-devmode project].
+
+For more details on dev mode with container support, see the documentation for the link:https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#devc-container-mode[devc goal of the Liberty Maven Plugin] or the link:https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md#libertydevc-task-container-mode[libertyDevc task of the Liberty Gradle Plugin].
+
+== Next steps
+
+As this is a tech preview, functionality can change and we are open to suggestions.  Give it a try and let us know what works or doesn't work for you.  You can post in the link:https://gitter.im/OpenLiberty/developer-experience[Open Liberty developer experience chat] or open issues in the link:https://github.com/OpenLiberty/ci.maven[Liberty Maven Plugin] or link:https://github.com/OpenLiberty/ci.gradle[Liberty Gradle Plugin] repositories.
```
Comment (yeekangc) [inline posts/2020-07-24-dev-mode-containers-preview.adoc]: Should we indicate that we look to iteratively provide additional previews for feedback?

**Comment by yeekangc** [comment]
Thank you, @ericglau.  Looks nice overall.

I am not sure we need the "Defining an application image" section since the actual instruction is only mentioned later.  The transition from it to the next section is somewhat odd IMHO too.  Also, should we refer to it as a "container image"?

File: posts/2020-07-24-dev-mode-containers-preview.adoc (hunk 1 above)
Comment (ericglau) [reply posts/2020-07-24-dev-mode-containers-preview.adoc]: Added call to action.  Will add demo gifs in a separate commit.

File: posts/2020-07-24-dev-mode-containers-preview.adoc (hunk 2 above)
Comment (ericglau) [reply posts/2020-07-24-dev-mode-containers-preview.adoc]: Added

**Comment by ericglau** [comment]
@yeekangc Updated as per your comments.

Live draft has been updated (and includes a gif): https://draft-openlibertyio.mybluemix.net/blog/2020/07/24/dev-mode-containers-preview.html

**Review by yeekangc — APPROVED** [review]
LGTM.  Thanks!

**Comment by ericglau** [comment]
@lauracowen Please review, thanks.

**Comment by Charlotte-Holt** [comment]
Closing PR to `staging` until final edits are made in `draft`.

**Pull request closed without merging**